.github/workflows/rust-ci.yml

```
@@ -160,12 +160,16 @@ jobs:
           find . -name Cargo.toml -mindepth 2 -maxdepth 2 -print0 \
             | xargs -0 -n1 -I{} bash -c 'cd "$(dirname "{}")" && cargo check --profile ${{ matrix.profile }}'
 
-      - name: cargo test
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: nextest
```

```suggestion
          tool: nextest
          version: 0.9.103
```

Let's pin the version to prevent any malicious injections and prevent any risks if ok for you?

---

Pinned version for `nextest` and `taiki-e/install-action`